.github/workflows/just-nuts.yml

```diff
@@ -18,6 +18,13 @@ jobs:
         repository:
           - salesforcecli/plugin-login
           - salesforcecli/plugin-settings
+          - salesforcecli/plugin-custom-metadata
+          - salesforcecli/plugin-community
+          - salesforcecli/plugin-env
+          - salesforcecli/plugin-generate
+          - salesforcecli/plugin-limits
+          # omitted while nuts are too flaky on bulk commands
+          # - salesforcecli/plugin-data
```

Comment:
I pulled this from `sfdx-cli`, is this still true? Plugin-data tests seem to have stabilized a bit (still not great) https://github.com/salesforcecli/plugin-data/actions/workflows/test.yml